src/main/java/com/github/webdriverextensions/newversion/FileExtractorImpl.java

```diff
@@ -14,6 +14,8 @@
 import java.nio.file.*;
 import java.util.regex.Pattern;
 
+import static jdk.nashorn.internal.objects.Global.println;
+
```

Comment:
These two lines should be removed again. The import is not needed / used.

Reply:
[I can not find them in the current code](https://github.com/webdriverextensions/webdriverextensions-maven-plugin/blob/919acd392fe7b3318a0badba728f62b6512b12d2/src/main/java/com/github/webdriverextensions/newversion/FileExtractorImpl.java). These lines are added by your commit. Importing an internal method from a deprecated package is also a bad idea.

Reply:
This was forked long ago. I never used this class in my life: it is too arcane.

Reply:
I do not see this line in my local version of the file. What's up with the merge logic?

Reply:
Ah, found that!